modules/ROOT/design-async-api.adoc

```diff
@@ -0,0 +1,126 @@
+= AsyncAPI Specifications   
+
```

Comment:
I suggest adding a lead-in here, something like:
You can design event-driven APIs using AsyncAPI in API Designer.

Reply:
Would you please go ahead and add the new page to the nav.adoc so we'll know where it'll appear in the TOC? Thank you!